packages/api-v2/src/hooks/useIsGrowthbookLoaded.ts

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect } from 'react';
+import { Analytics } from '@deriv-com/analytics';
+import useRemoteConfig from './useRemoteConfig';
+
+const useIsGrowthbookIsLoaded = () => {
```

Comment:
same as this!

Reply:
yes @deriv/hooks is not be used in wallets. As @deriv/hooks uses stores. So wallet team maintains a copy of all hooks in api-v2. This was added to api-v2 as per Nijil